src/jitlayers.cpp

```diff
@@ -851,10 +851,27 @@ class JLMemoryUsagePlugin : public ObjectLinkingLayer::Plugin {
                           jitlink::PassConfiguration &Config) override {
         Config.PostAllocationPasses.push_back([this](jitlink::LinkGraph &G) {
             size_t graph_size = 0;
+            size_t code_size = 0;
+            size_t data_size = 0;
```

Comment:
On aarch64 darwin I'm getting
```
/Users/mose/repo/julia/src/jitlayers.cpp:858:20: warning: variable 'data_size' set but not used [-Wunused-but-set-variable]
            size_t data_size = 0;
                   ^
/Users/mose/repo/julia/src/jitlayers.cpp:857:20: warning: variable 'code_size' set but not used [-Wunused-but-set-variable]
            size_t code_size = 0;
                   ^
```
(and currently we aren't using `-Werror` on CI on macOS)